tc_driver/Cases.jl

```diff
@@ -862,8 +933,14 @@ function initialize_profiles(
     prog_gm_uₕ = TC.grid_mean_uₕ(state)
     TC.set_z!(prog_gm_uₕ, prof_u, prof_v)
 
+    # TODO - no theta profile given
```

Comment:
you could use `T` given from the GATE paper directly in `dp_dz` right? 
what if we add `dp_dz` as a function in thermodynamics and have several versions of it these?

Reply:
done